CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ _Please add entries here for your pull requests that are not yet released._
 ### Changed
 
 - `--org` option now takes precedence over `CPLN_ORG` env var, which takes precedence over `cpln_org` from `controlplane.yml`. [PR 88](https://github.com/shakacode/heroku-to-control-plane/pull/88) by [Rafael Gomes](https://github.com/rafaelgomesxyz).
+- Deprecated `setup` entry in the ControlePlane config file and introduced `setup_app_templates` in its place. [PR 112](https://github.com/shakacode/heroku-to-control-plane/pull/112) by [Mostafa Ahangarhga](https://github.com/ahangarha).
```

Comment:
Typo:

```suggestion
- Deprecated `setup` entry in the Control Plane config file and introduced `setup_app_templates` in its place. [PR 112](https://github.com/shakacode/heroku-to-control-plane/pull/112) by [Mostafa Ahangarhga](https://github.com/ahangarha).
```

Also, I feel like "Renamed `setup` config to `setup_app_templates`." would be simpler and gets the point across.

Reply:
I think it would be good to mention deprecation somewhere in the changelog.